### DISCUSSION TITLE
Can I specify {{filename}} in additionalBranchPrefix?

### DISCUSSION THREAD

**Discussion by hidetatz** [renovatebot/renovate#16950] [Request Help]
We use renovate to update our GitHub Actions dependencies in our workflow files.
In our organization, we have multiple workflow files which use the same dependency in `.github/workflows` directory. Because each file is managed by different team, we want renovate to split PR based on the file even if it updates the same dependency.

After some googling I figured out I can use `additionalBranchPrefix` to order renovate to split the PR based on the file directory. However, because GitHub Actions workflow files must be located under single directory ( `.github/workflows` ), I don't think `additionalBranchPrefix: "{{baseDir}}-"` configuration works.

Is there any workaround for this issue? Or can I specify `filename` or something like this in additionalBranchPrefix?

**Comment by Gabriel-Ladzaretti** [comment]
have you tried `packageFile` and `packageFileDir`? full path `{{packageFileDir}}{{packageFile}}`, filename would be `{{packageFile}}`.
for additional exposed template string checkout https://docs.renovatebot.com/templates/#other-available-fields
